File: BoringApartments.cpp
#include <bits/stdc++.h>
using namespace std;

int main(void) {
    // variable declaration and input
    int t;
    cin >> t;
    vector<int> res;
    while (t--) {
        string x;
        cin >> x;
        if (x=="1") 
            res.push_back(1);
        else {
            int digit = x[0] - '0';
            int ans = (digit-1) * 10;
            for (int i=1; i<=x.size(); i++)
                ans += i;
            res.push_back(ans);  
        }
    }
    for (auto &x:res)
        cout << x << endl;

    return 0;
}

File: Python/BoringApartments.py
def main():
    t = int(input())
    res = []
    while t:
        t -= 1
        x = input()
        if x == "1":
            res.append(1)
        else:
            digit = ord(x[0])-ord('0')
            ans = (digit-1)*10
            for i in range(len(x)):
                ans += i
            res.append(ans)
    for i in res:
        print(i)


if __name__ == '__main__':
    main()
